### PR TITLE
[5/6] Venft owned tokens and summary

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Adds fetching for the user's open veNft positions, displays them in the Owned NFTs section, and calculates summary stats based on them.

You will need the following subgraph URL in order to enable this functionality:
`https://api.studio.thegraph.com/query/26890/jbx-rinkeby-w-vebanny/0.0.1`

Also note that the token images do not match the locked durations due to new options being set on the deployed contract. This will be resolved contract-side.

## Screenshots or screen recordings

Screen Shot 2022-07-23 at 9 36 15 AM

Screen Shot 2022-07-23 at 9 36 19 AM

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/veNft/VeNftOwnedTokenCard.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                    fullWords: true,
+                  })}
+                </p>
+                <p>
+                  {detailedTimeString({
+                    timeSeconds: remaining,
+                  })}{' '}
+                </p>
+              </Col>
+            </Row>
```
Comment (tomquirk) [inline src/components/veNft/VeNftOwnedTokenCard.tsx:104]: Antd `Descriptions` could be useful here.

File: src/components/veNft/VeNftOwnedTokenCard.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          </Col>
+          <Col span={4} />
+          <Col span={6}>
+            <Image src={thumbnailUri} alt="nft" preview={false} />
+          </Col>
+        </Row>
+        <Space direction="vertical" style={{ width: '100%' }}>
+          <Row>
+            <Button block onClick={() => setExtendLockModalVisible(true)}>
+              <Trans>EXTEND LOCK</Trans>
```
Comment (tomquirk) [inline src/components/veNft/VeNftOwnedTokenCard.tsx:115]: **question**: curious the reasoning behind all-caps. We might prefer "normal" capitalization unless theres a reason to not to do it.

File: src/components/veNft/VeNftSummaryStatsSection.tsx [hunk 3]
```diff
@@ -26,9 +38,16 @@ const VeNftSummaryStatsSection = ({
...
-          0
+          {totalStaked}
+        </Descriptions.Item>
+        <Descriptions.Item label={t`Total staked period`}>
+          {`${formattedTotalStakedPeriod} `}
+          <Plural
+            value={totalStakedPeriodInDays}
+            one={t`day`}
+            other={t`days`}
+          />
```
Comment (tomquirk) [inline src/components/veNft/VeNftSummaryStatsSection.tsx:49]: **praise** MY MAN

**Review by tomquirk — COMMENTED** [review]
@jmilldotdev this one's great. Cheers!

Created another follow-up issue for UI https://github.com/jbx-protocol/juice-interface/issues/1524

File: src/components/veNft/VeNftOwnedTokenCard.tsx (hunk 1 above)
Comment (jmilldotdev) [reply src/components/veNft/VeNftOwnedTokenCard.tsx:104]: Done

File: src/components/veNft/VeNftOwnedTokenCard.tsx (hunk 2 above)
Comment (jmilldotdev) [reply src/components/veNft/VeNftOwnedTokenCard.tsx:115]: Was copying from mockups, but agree it should be lower for consistency.

**Comment by jmilldotdev** [comment]
Addressed all.

**Pull request merged**